libr/asm/arch/pyc/opcode.c

```diff
@@ -0,0 +1,266 @@
+#include "opcode.h"
+
+pyc_opcodes *get_opcode_by_version(char *version) {
```

Comment:
this will be cleaner as a for loop over an array of structs containing version number andt he pointer to the opcode_ function.